libcudacxx/include/cuda/std/__type_traits/type_set.h

```diff
@@ -83,6 +83,10 @@ using __type_set_insert = decltype(__set::__bulk_insert::__call<_Ts...>(static_c
 template <class... _Ts>
 using __make_type_set = __type_set_insert<__type_set<>, _Ts...>;
 
+template <class _Ty, class... _Ts>
+_LIBCUDACXX_INLINE_VAR constexpr bool __is_contained_in =
```

Comment:
Not sure if I like this new name either. When the operation is on a type and a set then sure, but since this is an operation on a type and a pack, using sets as an implementation detail essentially, I think I'd rather have `is_included_in` or something in that vein. That's just a preference, but since this is about a "badly named" concept, I think it is a valid comment here.

Reply:
Yeah I am fine with `__is_included_in` I was just offended by it being named `_One_of` instead of `__one_of` and also doing more things than needed